tests/test_validation.py

```diff
@@ -0,0 +1,141 @@
+import os
+import pytest
+from pathlib import Path
+from parselglossy.validate import (validate_node,
+                                   check_predicates_node,
+                                   InputError,
+                                   TemplateError)
+from parselglossy.read_yaml import read_yaml_file
+from typing import Dict, Any
+
+
+JsonDict = Dict[str, Any]
+
+
+def _helper(category: str,
+            input_file_name: str,
+            template_file_name: str) -> JsonDict:
+    this_path = Path(os.path.dirname(os.path.realpath(__file__)))
```

Comment:
`Path(__file__)` is enough.

Reply:
Thanks for this - this is a major simplification and one more reason to go for `pathlib`.

Reply:
No - this was not correct since I need the directory, not the path. We can fix this later but for the moment I reverted my change.

Reply:
Agreed, probably it's `Path(__file__).parent` or some such.